### ISSUE TITLE
Run Shockwave Filter just once?

### ISSUE THREAD

**Issue by mspanish** [pixijs/pixi-filters#16]
I've tried setting the shockwave value to null using a settimeout and removing the filter, but that doesn't seem to work right. I want the shockwave applied just once to my bg sprite when the user clicks on it - not in a looping fashion like the sample  Is there a way to do this that is performance friendly?  Overall the filter is super cool but seems like it would be costly.

**Comment by englercj** [comment]
Not really any more expensive than other filters. Should just be one time as long as you don't loop the time uniform.

**Comment by mspanish** [comment]
Thanks that was just what I needed - I also added the shockwave code to my pixi particles emitter update() function, rather than calling a new one.  I assume it's bad to have multiple update() functions running, right? I also added a bit to use the current mouse value as the center, since I'm calling this when the child clicks/touches the underwater BG - 

I pass event.data with the position object to my fn, then use this (vw is view width for my app, vh is the view hight):

```
 var centerX = pos.x; 
 var decimalX = centerX/vw;
 var centerY = pos.y; 
 var decimalY = centerY/vw;
 var xPos = decimalX;
 var yPos = decimalY;
 shockwave.center = [xPos, yPos];
```

All looks lovely now - thanks a ton.
